drivers/storage/portworx/component/csi.go

```diff
@@ -530,6 +530,11 @@ func (c *csi) findPreinstalledCSISnapshotController() error {
 		return err
 	}
 	for _, pod := range podList.Items {
+		// Ignore csi pods deployed by operator
```

Comment:
can we change "app" into a constant and use it in csiDeploymentTemplateLabels initialization as well